sets/seaside.py

```diff
@@ -84,6 +85,72 @@ def duration(self):
 		self.game.announce('-- gaining +$1 and +1 action')
 
 
+class Lookout(crd.Card):
+	def __init__(self, game, played_by):
+		crd.Card.__init__(self, game, played_by)
+		self.title = "Lookout"
+		self.price = 3
+		self.description = "{} Look at the top 3 cards of your deck. Trash one of them. " \
+		                   "Discard one of them. Put the other one on the top of your deck.".format(crd.format_actions(1))
+		self.type = "Action"
+
+	@gen.coroutine
+	def play(self, skip=False):
+		crd.Card.play(self, skip)
+		self.played_by.actions += 1
+		self.game.announce('-- gaining 1 action')
+
+		topdeck_list = []
+		for i in range(3):
+			topdeck = self.played_by.topdeck()
+			if topdeck is not None:
+				topdeck_list.append(topdeck)
+
```

Comment:
`topdeck_list = list(filter(None.__ne__, [self.played_by.topdeck() for x in range(0,3)]))`